src/renderer/gdi/invalidate.cpp

```diff
@@ -54,7 +46,7 @@ HRESULT GdiEngine::InvalidateScroll(const COORD* const pcoordDelta) noexcept
 // - rectangles - Vector of rectangles to draw, line by line
 // Return Value:
 // - HRESULT S_OK or GDI-based error code
-HRESULT GdiEngine::InvalidateSelection(const std::vector<SMALL_RECT>& rectangles) noexcept
+HRESULT GdiEngine::InvalidateSelection(const std::vector<til::rect>& rectangles) noexcept

```

Comment:
selections used to be inclusive; has this semantically changed? are we sure all callers are doing the right thing?

Reply:
This is one of the cases of a `SMALL_RECT` with exclusive coordinates... 😅